Move all extensions to separate files

Pajk opened the pull request:
No real changes, just moved all extensions to separate files.

Pajk committed:
Move all extensions to separate files

VojtaStavik commented:
@Pajk Good idea. Thank you for all the work. Could you also update the curly braces formatting?

from:
func name()
{
      ///
}

to:
func name() {
     ////
}

VojtaStavik commented on extensions/UIColor+HesString.swift:
```diff
@@ -0,0 +1,66 @@
+public extension UIColor
```
There's a typo in the file name

Pajk committed:
A bit more consistent code formatting

+Fix typo in HexString extension

Pajk commented:
@VojtaStavik I changed the formatting and fixed the file name. Leaving it as a second commit because it wouldn't be easy to tell what was changed if I squashed it in the the first commit.
